lib/presentation/pages/on_boarding/onboarding.dart

```diff
@@ -0,0 +1,153 @@
+                      const SizedBox(
+                        height: 20,
+                      ),
+                      Expanded(
+                        child: Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: Text(
+                            contents[index].description,
+                            textAlign: TextAlign.center,
+                            style: const TextStyle(color: Colors.black),
+                          ),
+                        ),
+                      ),
+                      buildDots(index),
+                      Padding(
+                        padding: const EdgeInsets.all(15.0),
+                        child: Container(
+                            height: 60,
+                            width: double.infinity,
+                            decoration: BoxDecoration(
+                                borderRadius: BorderRadius.circular(15)),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                currentIndex == contents.length - 1
+                                    ? const SizedBox(
+                                        width: 25,
+                                      )
+                                    : TextButton(
+                                        onPressed: () =>
+                                            Get.offNamed(Routes.homeScreen),
+                                        child: const Text(
+                                          "Skip",
+                                          style: TextStyle(fontSize: 20),
+                                        ),
+                                      ),
+                                TextButton(
+                                    onPressed: () async {
+                                      if (currentIndex == contents.length - 1) {
+                                        Get.offNamed(Routes.homeScreen);
+                                      }
+
+                                      await controller.nextPage(
+                                          duration: const Duration(seconds: 1),
+                                          curve: Curves.easeIn);
+                                    },
+                                    child: Text(
+                                      currentIndex == contents.length - 1
+                                          ? "Continue"
+                                          : "Next",
+                                      style: const TextStyle(fontSize: 20),
+                                    )),
+                              ],
+                            )),
+                      )
+                    ],
+                  ),
+                )),
+              ],
+            );
+          },
+        ),
+      ),
+    );
+  }
+
+  Row buildDots(int index) {
```

Comment:
Make as a widget not as a Row you are using Row widget and taking return row too and best practice to make a different file for this widget.